.github/workflows/auto-approve-v2-merge-forward.yml

```diff
@@ -21,6 +21,9 @@ jobs:
       if: >
         github.event.pull_request.user.login == 'aws-cdk-automation'
         && github.event.pull_request.base.ref == 'v2-main'
-        && contains(github.event.pull_request.labels.*.name, 'pr/forward-merge')
```

Comment:
why not just create a label `pr/auto-merge` and apply it in both cases?

Reply:
Duh. yes, of course.